.gitpod.yml

```diff
@@ -1,24 +1,29 @@
 tasks:
-  - name: Deploy Jekyll Docs
-    command: export JEKYLL_ENV=production; cd /workspace/AaC/docs; bundle install && bundle exec jekyll s
+  - name: Serve Sphinx Docs
+    init: |
+      cd /workspace/AaC/docs/
```

Comment:
I didn't even think about the impact for GitPod when I made the switch and tried to clean up Jekyll references in workflows, which I missed some initially and tried to catch in my PR #658. Glad this got caught and corrected now.

Reply:
No worries!